static/styles/left_sidebar.css

```diff
@@ -199,33 +198,34 @@ li.show-more-topics {
 
 #private_messages_section {
     margin-bottom: $sections_vertical_gutter;
+    margin-right: 12px;
```

Comment:
we would not want the PM section to get completely attached to `message box view` and rather be at a `margin-right: 12px` to it, similar to how our `global filters` and `stream list` is.